content/en/docs/ambient/usage/l4-policy/index.md

```diff
@@ -75,6 +75,31 @@ Even though the identity of the pod is otherwise correct, the presence of a L7 p
 command terminated with exit code 56
 {{< /text >}}
 
+### Authorization policy attachment
+
+In the simplest enforcement scenario, you want to enforce policy against TCP attributes and you have no waypoint proxies in your traffic's path. These policies can be enforced by ztunnel proxies.
```

Comment:
I am still not comfortable with the hoops we have to jump through to describe "having a waypoint in your traffic's path"

https://istio.io/latest/docs/ambient/architecture/data-plane/ uses "in mesh" and "in mesh, waypoint enabled" which is close but probably still not how people are going to describe workload configuration.

Thoughts?

Reply:
Not sure the ideal way to word this if we don't like "no waypoint proxies in your traffic path" which I think is at least fairly clear, if somewhat clunky. I don't really like "waypoint disabled" personally but if that is the project's preferred wording I can change it.

Reply:
I've never heard "waypoint disabled" but I'd never use that; you don't define Stage 1 of something additive as "Without Stage 2" :)

"In the simplest enforcement scenario, you want to enforce policy against TCP attributes and you are only using the secure L4 overlay. These policies can be enforced by ztunnel proxies."

DIet Ambient™